.travis.yml

```diff
@@ -10,7 +11,17 @@ addons:
       - libtext-markdown-discount-perl
       - libtext-typography-perl
 
-script: make -C doc
+install:
+  - pip install ansible
+  - pip install ansible-lint
+  - ansible --version
+  - ansible-lint --version
+
+script:
+  - pushd ./ansible
+  - ansible-lint _lint_roles.yml
```

Comment:
I'd rather have a `ansible/lint.sh` script we can easily invoke outside CI too.
That way, we can test changes (to the Ansible code or to the lint script) without pushing & waiting for CI.

Reply:
yes sure i can do that.